pkgs/development/python-modules/aiohttp-zlib-ng/default.nix

```diff
@@ -36,7 +34,7 @@ buildPythonPackage rec {
   propagatedBuildInputs = [
     aiohttp
     zlib-ng
-  ] ++ lib.optional (lib.meta.availableOn stdenv.hostPlatform cpufeature) cpufeature;
```

Comment:
I mainly opened this, that the removal of the dependency here is not forgotten as it caused some trouble already.

Reply:
I'm aware, that adding it broke aarch64-linux builds of home-assistant, because aarch64 cpus have no CPUID support.